ddspipe_core/include/ddspipe_core/core/DdsPipe.hpp

```diff
@@ -304,7 +306,8 @@ class DdsPipe
      */
     void create_new_bridge_nts_(
             const utils::Heritable<types::DistributedTopic>& topic,
-            bool enabled = false) noexcept;
+            bool enabled = false,
+            const types::EndpointKind& endpoint_kind = types::EndpointKind::reader) noexcept;
```

Comment:
I don't think that's a good argument. If you code thinking you're smarter than the compiler, you'll end up losing. Let the compiler do whatever it will. I believe when you pass a const argument you are not going to modify, it's best to pass it as a reference. If, for whatever reason, with that kind there was a more efficient way, the compiler will figure it out. 

In any case, I'll remove the reference so you leave me alone.